autoload/ctrlp/bookmarks.vim

```diff
@@ -20,10 +20,15 @@ function! ctrlp#bookmarks#init() abort
         let l:line_nrs = sort(bm#all_lines(l:file), "bm#compare_lines")
         for l:line_nr in l:line_nrs
             let l:bookmark = bm#get_bookmark_by_line(l:file, l:line_nr)
+            if g:bookmark_show_detail_in_ctrlp
+              let l:detail=fnamemodify(l:file,':t').'|'.l:line_nr.'|'
```

Comment:
Actually as I see now in your implementation it will just show the content including all leading spaces/tabs. Instead it should alwas start right away with the content after one space. Could you please change that? Please check the default implementation for reference: https://github.com/MattesGroeger/vim-bookmarks/blob/master/autoload/bm.vim#L137 (though I couldn't find the place where the whitespace is trimmed right now).